Formula/a/agda.rb

```diff
@@ -70,7 +61,7 @@ def install
       system "cabal", "v2-update"
       system "cabal", "--store-dir=#{libexec}", "v2-install", *cabal_args, "--installdir=#{lib}/agda"
       system "./GenerateEverything"
-      system bin/"agda", "-i", ".", "-i", "src", "--html", "--vim", "README.agda"
+      system bin/"agda", "--large-indices", "-i", ".", "-i", "src", "--html", "--vim", "README.agda"
```

Comment:
With the newly released stdlib 1.7.3, there would be no need for `--large-indices`.